python_ta/cfg/visitor.py

```diff
@@ -12,14 +12,16 @@ class CFGVisitor:
         The top of the stack corresponds to the end of the list.
         (compound statement [while], {'Break'/'Continue': CFGBlock to link to})
     """
-    cfg: ControlFlowGraph
+    cfgs: List[ControlFlowGraph]
+    _current_cfg: ControlFlowGraph
```

Comment:
The type here should be `Optional[ControlFlowGraph]`. Similarly, the `_current_block` attribute should also be `Optional`.